addons/pos_self_order/static/src/LandingPageComponents/LandingPage/LandingPage.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+    <t t-name="LandingPage" owl="1">
+        <div class="o_so_landing_page d-flex flex-column h-100 justify-content-between align-items-center p-4 overflow-auto"
+            t-attf-style="
+                background-image: linear-gradient(to bottom, transparent 33%, #000000 99%), url(/menu/get-bg-image?pos_id={{selfOrder.config.pos_id}});
```

Comment:
I believe there is already a way to fetch image. The endpoint looks like this:
```js
`/web/image?model=product.product&field=image_128&id=${product.id}&unique=${product.write_date}`
```
Check if you can use it.

Reply:
for the image controller, i think i might need to be logged in:  ```odoo.addons.base.models.ir_model: Access Denied by ACLs for operation: read, uid: 4, model: product.product```